### PR TITLE
changes in folding

### PR DESCRIPTION
**Pull request by geethanjalivimal**

### CODE REVIEW

File: vedic_maths/folding.py [hunk 1]
```diff
@@ -223,14 +229,14 @@ def construct(self):
         ]
         e = Explanation(self, text, wait=2, fade=True, aligned_edge=LEFT)
 
-        """ text = [
+        text = [
```
Comment (kmadathil) [inline vedic_maths/folding.py:232]: This is confusing, since we don't have an example; that is why I commented it out

Comment (geethanjalivimal) [reply vedic_maths/folding.py:232]: Noted.

Comment (kmadathil) [reply vedic_maths/folding.py:232]: Please push this change and I will merge. Everything else is fine.

Comment (geethanjalivimal) [reply vedic_maths/folding.py:232]: Pushed it. Not able to create a new pull request.

**Pull request merged**